packages/flutter/test/widgets/gesture_detector_semantics_test.dart

```diff
@@ -76,4 +77,484 @@ void main() {
 
     semantics.dispose();
   });
+
```

Comment:
I believe this is missing some tests that show that the actions specified in a custom delegate actually show up in the semantics tree. (the tests above should cover this for the default delegate).

Reply:
Done